features/luks/data/lib/uevent/handlers/085-luks

```diff
@@ -83,12 +101,17 @@ read_pkcs11_key() {
 	output_file="$path"
 	flags="-l -r"
 	if plymouth --ping 2>/dev/null; then
-		add_new_line="true"\
-		plymouth ask-for-password \
-                                --prompt "Please enter passphrase for smart card:" \
-                                --number-of-tries=3 \
-                                --command="pkcs11-tool-wrapper"
-		rc=$?
+		if unl0kr -V 2>/dev/null; then
```

Comment:
I don't like this check of unl0kr feature, it should be implemented other way. @legionus what do you think?

Reply:
Since we have bash, it would be more native to write:
```shell
if type -P unl0kr >/dev/null; then
```
This does not require running the utility.  `plymouth --ping` is executed because the splash screen is checked to be active.